osidb/models.py

```diff
@@ -606,6 +607,13 @@ def fts_search(q):
         # If search has no results, this will now return an empty queryset
 
 
+@pghistory.track(
+    pghistory.InsertEvent(),
+    pghistory.UpdateEvent(),
+    pghistory.DeleteEvent(),
+    exclude="local_updated_dt,meta_attr,_alerts",
```

Comment:
The `_alerts` attribute will unfortunately soon cause some conflict as Daniel is just reworking it into a proper model. That future model can also be excluded from the history as we can recreate the alerts if needed.